samples/matter/lock/src/credentials/credentials_data_types.h

```diff
@@ -8,6 +8,7 @@
 
 #include <app/clusters/door-lock-server/door-lock-server.h>
 #include <lib/core/ClusterEnums.h>
+#include <type_traits>
```

Comment:
Why is this one needed?

Reply:
It is not needed anymore, thanks for catching it.